apps/src/code-studio/pd/form_components/FormController.story.jsx

```diff
@@ -156,6 +164,14 @@ export default storybook => {
 
   storybook
     .storiesOf('FormController', module)
+    .addDecorator((story) => (
+      // Currently the Bootstrap 3 styles required by React-Bootstrap are only applied inside div#application-container.
+      // This is to prevent conflicts with other parts of Code Studio using Bootstrap 2.
+      // See pd.scss. Without this container div it won't render properly.
+      <div id="application-container">
+        {story()}
+      </div>
+    ))
```

Comment:
I might just drop it into a function within this file for now, and possibly migrate it to a more global utils file as we (hopefully) expand our usage of react-bootstrap in stories